sql/src/main/java/io/crate/execution/engine/aggregation/AggregateCollector.java

```diff
@@ -60,12 +62,14 @@ public AggregateCollector(List<? extends CollectExpression<Row, ?>> expressions,
                        AggregationFunction[] aggregations,
                        Version indexVersionCreated,
                        BigArrays bigArrays,
-                       Input[]... inputs) {
+                       Input[][] inputs,
+                       Input[] filterInputs) {
```

Comment:
```suggestion
                       Input[] filterPerAggregate) {
```

To clarify that these correlate with the aggregations (though not sure if it does, maybe you've a better idea)

Reply:
`aggregateFilterInputs`? but it is used everywhere in the context of an aggregate (`AggregateContext`, `Aggregation`, `AggregateCollector`), imho it is only not that clear in `GroupingCollector`.

Reply:
Maybe just `filters` then? I think the `Inputs` can be dropped, it's visible from the type and imho doesn't add any useful information

Reply:
👍 will go with `filters`